spec/features/cases/offender_sar_complaint/case_viewing_spec.rb

```diff
@@ -15,23 +15,41 @@
 
     click_link("My open cases")
 
-    expect(page).to have_content("In time (5)")
+    expect(page).to have_content("In time (7)")
+    # expect(page).to have_css('span.moj-badge.moj-badge--red', text: "High priority")
     expect(page).to have_content("Late (4)")
 
     open_cases_page.case_filters.filter_cases_link.click
     open_cases_page.case_filters.filter_complaint_type_link.click
+
     open_cases_page.filter_complaint_type_content.complaint_litigation_checkbox.click
     open_cases_page.case_filters.apply_filters_button.click
 
     expect(page).to have_content("In time (2)")
     expect(page).to have_content("Late (3)")
   end
 
+  scenario "Filter cases 'Filter by priority' - Priority High" do
```

Comment:
The filtering here seems unnecessary. Don't you just want to load the cases page when there is a high priority case, and make sure the flag appears?